deepface/detectors/DetectorWrapper.py

```diff
@@ -123,10 +123,14 @@ def detect_faces(
             aligned_img, angle = detection.align_face(
                 img=img, left_eye=left_eye, right_eye=right_eye
             )
-            x1_new, y1_new, x2_new, y2_new = rotate_facial_area(
-                facial_area=(x2, y2, x2 + w2, y2 + h2), angle=angle, direction=1, size=img.shape
+            rotated_x1, rotated_y1, rotated_x2, rotated_y2 = rotate_facial_area(
+                facial_area=(x, y, x + w, y + h),
```

Comment:
we want to rotate the expanded facial area. instead x, you should pass x2 similar to previous implementation.

Reply:
Oh ok. You're right.
Forgot I amended the caller too. 
Doing it now